src/main/java/com/google/sps/data/BlurImage.java

```diff
@@ -33,6 +36,12 @@ public BlurImage(
     this.blobKeyString = blobKey.getKeyString();
     this.jsonBlurRectangles = jsonBlurRectangles;
     this.dateCreated = dateCreated;
+
+    BlobInfo blobInfo = new BlobInfoFactory().loadBlobInfo(blobKey);
+    // getSize() function returns the size in bytes. We divide by 1024 to
+    // convert it in KB. We can cast it to int because it can't be bigger than
+    // 8192.
+    sizeInKB = (int) (blobInfo.getSize() / 1024);
```

Comment:
Nit: technically it's KiB (kibibyte https://en.wikipedia.org/wiki/Kibibyte since you're dividing by 1024).

Reply:
Solved, thanks!